streamtrace.hh

```diff
@@ -229,6 +232,19 @@ struct capability_register
 static_assert(sizeof(capability_register) <= 32,
               "Capability register structure has grown far too big!");
 
+union debug_trace_register
```

Comment:
It would probably be cleaner for Python to add accessors (which would check that the register is the correct type), rather than exposing the union directly.